src/modules/back-to-top/back-to-top.component.html

```diff
@@ -0,0 +1,3 @@
+<div #backToTop class="back-to-top" (click)="goToTop()" [ngStyle]="{'background-color': backgroundColor, 'color': color}">
```

Comment:
Stache Components rely on SKY UX styles, so in a sense they require SKY UX. I don't see a situation where Stache Components would be used outside of SKY UX SPA. Also, we're not providing color configuration anywhere else, so it seems a bit out of place. 

While having adaptable styles is forward-thinking in some respects, it adds complication and inconsequential configuration to something that should be relatively simple and prescriptive. If this feature is asked for in the future by multiple teams, it might be worth considering. Even still, if they wanted to style the component differently, we're exposing the classnames on the front end (`.stache-back-to-top`).

We should develop with extensibility in mind, but not to add features for the sake of having features.